core/main.go

```diff
@@ -154,7 +203,22 @@ func main() {
 		fatalIfErr(err, "Import policy failure")
 	}
 
+	// Create Admin token associated with admin policy in Vault
+	lc.Info("Creating Vault Admin token.")
+	err = createToken(config.SecretService.TokenName4Admin, config.SecretService.PolicyName4Admin, rootToken.Token, config, client)
+	if err != nil {
+		lc.Error("Fatal Error creating Admin token in Vault.")
+		fatalIfErr(err, "Create token failure (Admin)")
+	}
+
+	// ------------------ Kong Vault Policies and associated token ----------------------
 	// Read the Kong HCL file and build the policy request
+	lc.Info("Verifying Kong policy file hash (SHA256).")
+	policyFile = config.SecretService.PolicyPath4Kong
+	_, err = hashFile(&policyFile)
```

Comment:
why is the policy file hashed at all here?

Reply:
@anonymouse64 I want to fingerprint compare the HCL files before they are imported, integrity/security. This is the first part of the new feature I want to implement. For now it's there to help out as a message, enforcing discrepancies will be next. I'm also testing digital signature of the fingerprint using Vault.

Reply:
I see so it's just that the feature isn't complete yet